example/web/index.html

```diff
@@ -34,7 +34,7 @@
        application. For more information, see:
        https://developers.google.com/web/fundamentals/primers/service-workers -->
   <script>
-    var serviceWorkerVersion = null;
+    var serviceWorkerVersion = '{{flutter_service_worker_version}}';
```

Comment:
This is no longer a valid script tag as of Flutter 3.2X 


```html
...
  <link rel="manifest" href="manifest.json" />
  </head>
  <body>
    <script src="flutter_bootstrap.js" async></script>
  </body>
```

Reply:
fixed